stdlib/InteractiveUtils/src/macros.jl

```diff
@@ -32,7 +32,7 @@ function gen_call_with_extracted_types(__module__, fcn, ex0)
     ex = Meta.lower(__module__, ex0)
     exret = Expr(:none)
     if !isa(ex, Expr)
-        exret = Expr(:call, :error, "expression is not a function call or symbol")
+        return Expr(:call, :error, "expression is not a function call or symbol")
```

Comment:
Let's just separate this `if` from the rest of the `elseif` chain.  E.g.:

```julia
if !isa(ex, Expr)
    return Expr(:call, :error, "expression is not a function call or symbol")
end

if ex.head == :call
....
```

That makes it a little clearer that this `if` statement is conceptually at a slightly different level than the next couple of `if` statements.